psutil/_psosx.py

```diff
@@ -212,6 +213,137 @@ def disk_partitions(all=False):
 # =====================================================================
 
 
+def sensors_temperatures():
+    """Return CPU temperatures including key name and temperature.
+
+    References:
+
+    https://stackoverflow.com/questions/28568775/
+        description-for-apples-smc-keys/31033665#31033665
+
+    https://github.com/Chris911/iStats/blob/
+        09b159f85a9481b59f347a37259f6d272f65cc05/lib/iStats/smc.rb
+
+    http://web.archive.org/web/20140714090133/http://www.parhelia.ch:80/
+        blog/statics/k3_keys.html
+    """
+    ret = collections.defaultdict(list)
+    temperatures = (
+        # group, key, label
+
+        # --- CPU
+        ("CPU", "TCXC", "PECI CPU"),
+        ("CPU", "TCXc", "PECI CPU"),
+        ("CPU", "TC0P", "CPU 1 Proximity"),
+        ("CPU", "TC0H", "CPU 1 Heatsink"),
+        ("CPU", "TC0D", "CPU 1 Package"),
+        ("CPU", "TC0E", "CPU 1"),
+        ("CPU", "TC0F", "CPU 1"),
```

Comment:
Judging from the script output you paster earlier it seems we get a duplicate for TC0E and TC0F so I suggest to remove TC0F